models/core/final/core__condition.sql

```diff
@@ -510,24 +510,33 @@ where diagnosis_code_25 is not null
 )
 
 select distinct
-  eg.encounter_id,
-  unpivot_cte.claim_id,
-  unpivot_cte.patient_id,
-  unpivot_cte.condition_date as condition_date,
-  unpivot_cte.condition_type as condition_type,
-  unpivot_cte.code_type as code_type,
-  unpivot_cte.code as code,
-  icd.description as description,
-  unpivot_cte.diagnosis_rank as diagnosis_rank,
-  unpivot_cte.present_on_admit_code as present_on_admit_code,
-  poa.present_on_admit_description as present_on_admit_description,
-  unpivot_cte.data_source as data_source,
-  '{{ var('last_update')}}' as last_update
+    null as condition_id,
+    unpivot_cte.patient_id,
+    eg.encounter_id,
+    unpivot_cte.claim_id,
+    unpivot_cte.condition_date as recorded_date,
+    null as onset_date,
+    null as resolved_date,
```

Comment:
What If someone has these fields and maps to a staging layer then we null out here. Is that how we want this to work?

Reply:
No, we don't want to hardcode the NULLs but my focus in this PR was just to make Core data model changes.  I didn't go back and remap these fields because I will be working on the LDS connector again and making any Tuva Project changes needed for that so I was going to re-do these mappings when I tackle that work.